app/_hub/kong-inc/app-dynamics/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+name: AppDynamics
+publisher: Kong Inc.
+version: 3.1.0
+desc: Integrate Kong with the AppDynamics APM Platform
+description: |
+  Report requests proxied by Kong to the AppDynamics APM Platform.
+enterprise: true
+type: plugin
+categories:
+  - analytics-monitoring
+kong_version_compatibility:
+  community_edition:
+    compatible: null
+  enterprise_edition:
+    compatible: true
+params:
+  name: app-dynamics
+  service_id: true
+  route_id: true
+  consumer_id: true
+  konnect_examples: false
+  protocols:
+    - http
+    - https
+  dbless_compatible: 'yes'
+
+---
+
+# Kong Plugin for AppDynamics
+
+This plugin integrates Kong with the AppDynamics APM platform so that
+proxy requests handled by Kong can be identified and analyzed in
+AppDynamics.  The plugin reports request and response timestamps as
+well as error information to the AppDynamics platform so that they can
+be analyzed in the AppDynamics flow map and correlated with other
+systems participating in handling application API requests.
+
+The plugin utilizes the
+[AppDynamics C/C++ Application Agent and SDK](https://docs.appdynamics.com/pages/viewpage.action?pageId=42583435),
+which must be downloaded and installed on the machine or in the
+container that is running Kong Gateway.  Please refer to the
```

Comment:
Does both control plane and data plane need the `AppDynamics C/C++ Application Agent and SDK` ?